tests/test_run_state_manager.py

```
@@ -0,0 +1,40 @@
+    def setUp(self):
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.runs_root = Path(self.temp_dir.name)
+        self.manager = RunStateManager(self.runs_root)
+
+    def tearDown(self):
+        self.temp_dir.cleanup()
+
+    def test_list_runs_async_calls_list_runs(self):
+        """Test that list_runs_async calls list_runs."""
+        with patch.object(self.manager, "list_runs", return_value=[]) as mock_list_runs:
+            async def run_test():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)